controllers/v1/observationsController.js

```diff
@@ -606,6 +630,207 @@ module.exports = class Observations extends Abstract {
     });
   }
 
+  async searchEntities(req) {
+    return new Promise(async (resolve, reject) => {
+      try {
+        let formatForSearchEntities = true;
```

Comment:
@borkarsaish65 instead of doing all the logical operation in observation controller this should be moved to a new helper function.

Reply:
its an API @VISHNUDAS-tunerlabs

Reply:
done